event-grid/subscribe-to-blob-storage/subscribe-to-blob-storage.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+# You must have the latest version of the Azure CLI
+
 # Provide a unique name for the Blob storage account.
 storageName=contosostorage
```

Comment:
Storage name needs to be unique yet it isn't formatted the same as other user-supplied values. Change to be consistent.